not-to-release/sources/answers/20111108104551AAVAVQR_ans.xml.conllu

```diff
@@ -348,17 +348,17 @@
 3	H	H	NOUN	NN	Number=Sing	6	nsubj	6:nsubj	_
 4	should	should	AUX	MD	VerbForm=Fin	6	aux	6:aux	_
 5	be	be	AUX	VB	VerbForm=Inf	6	cop	6:cop	_
-6	6.5	6.5	NUM	CD	NumType=Card	0	root	0:root	SpaceAfter=No
+6	6.5	6.5	NUM	CD	NumForm=Digit|NumType=Frac	0	root	0:root	SpaceAfter=No
```

Comment:
As I read the [NumType page](https://universaldependencies.org/en/feat/NumType.html), `NumType=Frac` is really limited to fraction words like "half". This should be `Card` I think.

Reply:
That's fair.  I was following the GUM standard, which marks them as `Frac`.  I will change them here and then file an issue with GUM, I suppose

```
29      =       =       SYM     SYM     _       26      parataxis       26:parataxis    _
30      7.64    7.64    NUM     CD      NumForm=Digit|NumType=Frac      29      xcomp   29:xcomp        Entity=(165-abstract-giv:act-cf3-1,3-coref
31      ±       ±       CCONJ   CC      _       32      cc      32:cc   _
32      1.12    1.12    NUM     CD      NumForm=Digit|NumType=Frac      30      conj    29:xcomp|30:conj:plus_minus     Entity=165)|SpaceAfter=No
```

@amir-zeldes

Reply:
https://github.com/UniversalDependencies/UD_English-EWT/pull/336/commits/475ea1f7def58be7051c25bdbb6ff0342e57cf9a